File: common/src/test/java/dev/number6/generate/ChannelComprehensionSummaryGenerator.java
package dev.number6.generate;


import dev.number6.db.model.ChannelComprehensionSummary;
import uk.org.fyodor.generators.Generator;

import java.time.LocalDate;

public class ChannelComprehensionSummaryGenerator implements Generator<ChannelComprehensionSummary> {

    @Override
    public ChannelComprehensionSummary next() {
        return new ChannelComprehensionSummary(CommonRDG.string().next(), LocalDate.now());
    }
}

File: comprehend/src/test/kotlin/dev/number6/generate/ChannelComprehensionSummaryGenerator.kt
package dev.number6.generate

import dev.number6.db.model.ChannelComprehensionSummary
import uk.org.fyodor.generators.Generator
import uk.org.fyodor.generators.RDG
import java.time.LocalDate

class ChannelComprehensionSummaryGenerator : Generator<ChannelComprehensionSummary> {
    override fun next(): ChannelComprehensionSummary {
        return ChannelComprehensionSummary(RDG.string().next(), LocalDate.now())
    }
}